hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -384,26 +388,5 @@ sudo apt full-upgrade
 * Install the `stable` version of the bootloader and replace the current configuration settings to enable USB boot.
 See `BOOT_ORDER` property if you wish to migrate the configuration by hand.
 ```
-sudo rpi-eeprom-update -d -f /lib/firmware/raspberrypi/bootloader/stable/pieeprom-2020-06-15.bin
-```
-Alternatively, use the Raspberry Pi Imager to create a custom SD card from the lastest MSD STABLE image on the [releases](https://github.com/raspberrypi/rpi-eeprom/blob/master/releases.md) page.
-
-* Reboot and check the bootloader version and config:
-```
-vcgencmd bootloader_version 
-vcgencmd bootloader_config
+sudo rpi-eeprom-update -d -f /lib/firmware/raspberrypi/bootloader/stable/pieeprom-2020-07-31.bin
```

Comment:
Assuming the user has set `/etc/default/rpi-eeprom-update` to `stable`, do they need to manually specify the specific eeprom filename?

Reply:
Switched to -a -d. Most people will want the factory defaults and not specifying that risks propagating a config with BOOT_ORDER set to something which doesn't support USB.
That would only impact people who have previously customized the boot e.g. for network boot